pyteal/ast/multi_test.py

```diff
@@ -1,6 +1,8 @@
 from typing import List
 
-from .. import *
+from pyteal import *
+from pyteal.ast import *
+from pyteal.ir import *
```

Comment:
Why did this change? The convention in this repo is to use relative imports, and there should be no reason to directly import the `ast` and `ir` modules

Reply:
👍